test/units/TEST-50-DISSECT.dissect.sh

```diff
@@ -516,6 +516,72 @@ rm -rf "$VDIR" "$EMPTY_VDIR"
 systemd-dissect --umount "$IMAGE_DIR/app0"
 systemd-dissect --umount "$IMAGE_DIR/app1"
 
+# Check reloading refreshes vpick extensions
+VBASE="vtest$RANDOM"
+VDIR="/tmp/${VBASE}.v"
+mkdir "$VDIR"
+cat >/run/systemd/system/testservice-50g.service <<EOF
+[Service]
+Type=notify-reload
+EnvironmentFile=-/usr/lib/systemd/systemd-asan-env
+ExtensionDirectories=${VDIR}
```

Comment:
so, please add tests that use this in combination with RootDirectory= and RootImage=. I am pretty sure neither of those codepaths work right now.

Reply:
Yes this is a good idea. It does work right now, in the sense that nothing explodes, but the wrong /etc/ is going to be added, so what the test needs to do is explicitly check for something that is in the RootImage but not on the host's, to ensure the feature works as expected.